### ISSUE TITLE
FreeBSd 14.1-RELEASE-p4 port build (and package) fails.

### ISSUE THREAD

**Issue by jimmyb-gh** [dreibh/tsctp#2]
Lots of notes about how to build the program, NO notes on how to use it.

Can you please update the README with several examples, and samples of expected output?

Also, the current FreeBSD implementation seems broken.
uname -a:
FreeBSD jpbsd 14.1-RELEASE-p4 FreeBSD 14.1-RELEASE-p4 GENERIC amd64

Running with and without sudo results in:

[jpb@jpbsd:/usr/ports/net/tsctp/work/stage/usr/local/bin]$ sudo ./tsctp 
socket: Protocol not supported
ipv6only: Bad file descriptor
setsockopt: Bad file descriptor
bind: Bad file descriptor
listen: Bad file descriptor
set event failed: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
accept: Bad file descriptor
^C

Jim B.

**Comment by dreibh** [comment]
Indeed, examples were missing. There is now an improved readme file in Git: https://github.com/dreibh/tsctp/blob/master/README.md . The tsctp manpage has also been improved significantly. I will soon create a new release and update the port.

Note that the output above looks like the SCTP kernel module is not loaded (check /boot/loader.conf for sctp_load="YES" to enable SCTP when booting). I also added this information to readme file and manpage.

**Comment by dreibh** [comment]
The FreeBSD port is up-to-date now.
